ui-ngx/src/app/modules/home/components/alarm/alarm-comment.component.html

```diff
@@ -69,10 +69,14 @@
                *ngIf="!displayDataElement.edit; else commentEditing"
                (mouseenter)="onCommentMouseEnter(displayDataElement.commentId, i)"
                (mouseleave)="onCommentMouseLeave(i)">
-            <div class="user-avatar" fxLayout="row" fxLayoutAlign="center center" fxFlexAlign="start" fxHide.xs
+            <div *ngIf="displayDataElement.userExists; else userDeleted" class="user-avatar"
+                 fxLayout="row" fxLayoutAlign="center center" fxFlexAlign="start" fxHide.xs
                  [style.background-color]="displayDataElement.avatarBgColor">
               {{ getUserInitials(displayDataElement.displayName) }}
             </div>
+            <ng-template #userDeleted>
+              <mat-icon matPrefix class="mat-icon tb-mat-30" fxHide.xs>no_accounts</mat-icon>
```

Comment:
Why is the size 30px and not 28px as in the `.user-avatar` class? Why add `matPrefix`?